README.md

```diff
@@ -17,4 +17,169 @@
 - Booleans, Comparison Operators, Flow Control 
 - Arrays
 
+
+## Strolling through the zoo
+Imagine you're walking in the zoo. Now, take a look around you. Aren't those lions, parrots and elephants awesome? Wouldn't it be cool if, in our code, we could catalogue the animals we see? To do that, we need to take a moment to talk about data structures. Specifically, how we would store that data in an **object**.
+
+Let's use the lion as an example. Try and describe it in your head. Think of any relevant information we might care about. We can come up with several properties for the lion: the weight, its age, its name, and so on. Wait a minute; let's back up to that previous sentence real quick — there's an important word there. *Properties.* An object is basically a bunch of properties grouped together, kind of like a filing cabinet.
+
+A property has a name (a *key*) that is connected to a value. For example, our lion object would have a `weight` key, which has a value of `190` (thanks, Google!). Of course, we're using kilograms here to express the lion's weight, since the metric system is *clearly* the superior system — European bias notwithstanding.
+
+
+## A data structure fit for a king
+The most common way to create an object is to use the *object literal* notation, as shown here:
+
+```js
+const animal = {};
```

Comment:
I <3 `const`, but keep in mind that this will prevent students from coding along very well, as students won't be able to redeclare the variables here.